athena_cfn/athena-cfn.yml

```diff
@@ -0,0 +1,146 @@
+# Purpose:  Enable Athena functionality from Quilt web catalog
+#===============================================================================
+# aws cloudformation validate-template --template-body file://athena/athena_cfn.yml
+#  Scoped to ${AWS::StackName}
+
+AWSTemplateFormatVersion: "2010-09-09"
+Description: |
+  Enable Athena from Quilt - Creates Athena Workgroup, Bucket, Policy
+
+Parameters:
+  DatabaseName:
+    Default: default
+    NoEcho: false
+    Description: Name of an EXISTING Athena database to use for Quilt tables
+    Type: String
+    MinLength: 1
+    MaxLength: 16
+    AllowedPattern: '[a-z][_a-z]*'
+  DatabaseTableRegex:
+    Default: '*_quilt_*'
```

Comment:
That's not regex.

Reply:
Sorry, are you saying it won't work, or just that I should not call that a Regex?

Reply:
> or just that I should not call that a Regex?

this